src/Annotator.js

```diff
@@ -57,9 +57,24 @@ class Annotator extends EventEmitter {
         this.annotationModeHandlers = [];
         this.localized = options.localizedStrings;
 
-        const { file } = this.options;
+        const { apiHost, file, token } = this.options;
         this.fileVersionId = file.file_version.id;
         this.fileId = file.id;
+
+        this.getAnnotationPermissions(this.options.file);
```

Comment:
This would be more clear if it returned something that we set `this.annotationsPermissions` to. Avoids the problem of having a seemingly random class variable created outside of the constructor.

Reply:
👍